examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientTls.java

```diff
@@ -108,27 +109,16 @@ public static void main(String[] args) throws Exception {
         }
 
         HelloWorldClientTls client;
-        switch (args.length) {
-            case 2:
-                client = new HelloWorldClientTls(args[0], Integer.parseInt(args[1]),
-                        buildSslContext(null, null, null));
```

Comment:
I think we still want this case. Using the default ca is good for tls.

Reply:
This case will not work, with `PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target`.

Reply:
@ejona86 I think that usecase is for advanced users who are experienced with TLS. For the first time learners, they still couldn't run that usecase successfully without extra guidance. For advanced users, they mostly like would know how to use default ca for their own project even this example doesn't show that option. So adding this option only adds complexity and distract first time learners.

Reply:
With the test server certificate it is a failure. But with a real server certificate, it would work. We want to encourage people to use the default ca when possible.

Reply:
Ok, I put back the use case of using default CA and put a note on it.